docs/documentation/new.md

```diff
@@ -119,15 +122,25 @@ Default applications are created in a directory of the same name, with an initia
   <p>
     The name of the source directory.
   </p>
+  <p>
+    You can change the value in <em>.angular-cli.json</em> (<code>apps[0].root</code>).
+  </p>
 </details>
 
 <details>
   <summary>style</summary>
   <p>
     <code>--style</code> <em>default value: css</em>
   </p>
+  <div>
+    The style file default extension. Possible values:
+    <ul>
+      <li>css</li>
+      <li>scss</li>
```

Comment:
as far as I can see, from `commands/new.ts` up until `blueprints/ng/index.ts`, there's only validation for `stylus` to change the extension to `styl`. Meaning that anyone can f.e put `asdf` and that would work. I also don't see any e2e testing for that.